src/services/actions.ts

```diff
@@ -17,21 +18,23 @@ import { ReconnectableActions } from './uml-relationship/reconnectable/reconnect
 import { UMLRelationshipActions } from './uml-relationship/uml-relationship-types';
 import { UndoActions } from './undo/undo-types';
 
-export type Actions = EditorActions &
-  LayouterActions &
-  UMLContainerActions &
-  UMLElementActions &
-  UMLRelationshipActions &
-  ConnectableActions &
-  ReconnectableActions &
-  HoverableActions &
-  InteractableActions &
-  MovableActions &
-  MovingActions &
-  ResizableActions &
-  ResizingActions &
-  SelectableActions &
-  UpdatableActions &
-  AssessmentActions &
-  UndoActions &
-  CopyActions;
+export type Actions =
```

Comment:
Instead of composing a type consisting of all actions merged together, we need and either-or type representing any valid action type.